src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,44 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
```

Comment:
Please destruct movie object
```suggestion
export const MovieCard = ({ movie }) => (
  const {
    title,
    description,
    ...
  } = movie
```